test_common/harness/parseParameters.h

```diff
@@ -19,6 +19,11 @@
 #include "compat.h"
 #include <string>
 
+enum ExecutionMode
+{
+    kExecute = 0,
+    kGenerate ,
+};
```

Comment:
Couldn't this be another compilation mode instead? `kGenerateSource`?

Reply:
I am okay with adding this as another compilation mode - will follow up on this